schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -2385,6 +2385,10 @@
           "type": "boolean",
           "description": "If true, components needed for interacting with the Kubernetes API with OIDC authentication (Gangplank, Dex) be deployed and configued."
         },
+        "trustedCAPath": {
+          "type": "boolean",
+          "description": "If true, the path to a root CA to trust for self-signed certificates at the Oauth2 URL will be defined."
```

Comment:
I don't think this is correct, the path to the ca.crt is hardcoded in the template to `/tls/ca.crt`.

That file should contain the (self-signed) CA certificate of the oauth2 endpoint so Gangplank can trust it.